fabric/src/main/resources/minescript.py

```diff
@@ -786,6 +786,53 @@ def screen_name():
   """
   return CallScriptFunction("screen_name")
 
+def container_get_item(slot:int):
+  """Gets the item in a container slot if there is one.
+
+  Args:
+    slot: slot number
+
+  Returns:
+    `None` if there is no item in that slot, no inventory is open or an invalid inventory is open.
+    dictionary of item data if there is an item in that slot.
+
+  Since: v3.3
+  """
+  return CallScriptFunction("container_get_item", slot)
+
+def container_get_items():
+  """Gets all items in a container.
```

Comment:
Please document the type of GUI screen that needs to be visible for items to be returned, and what value is returned (or exception raised) if there is no such GUI screen.